db/db.go

```diff
@@ -48,8 +48,8 @@ func NewClient(ctx context.Context, c *internal.DatabaseConfig) (*Client, error)
 		return nil, err
 	} else if p.Scheme != "https" {
 		return nil, fmt.Errorf("invalid database URL: %q; want scheme: %q", c.URL, "https")
-	} else if !strings.HasSuffix(p.Host, ".firebaseio.com") {
-		return nil, fmt.Errorf("invalid database URL: %q; want host: %q", c.URL, "firebaseio.com")
+	} else if !(strings.HasSuffix(p.Host, ".firebaseio.com") || strings.HasSuffix(p.Host, ".firebasedatabase.app")) {
```

Comment:
I think going forward we want to remove this domain validation entirely. That's what we did in our Python SDK. Can we change this PR to just drop the domain validation?

Reply:
Done